release-tools/pom.xml

```diff
@@ -103,6 +103,19 @@
 			<version>0.9.1</version>
 		</dependency>
 
+		<dependency>
+			<groupId>com.github.tomakehurst</groupId>
+			<artifactId>wiremock</artifactId>
+			<version>1.58</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
```

Comment:
`spring-shell` and `wiremock`. `spring-shell` brings Guava 17 but wiremock needs at least v18